docs/docusaurus.config.js

```diff
@@ -134,6 +134,14 @@ module.exports = {
           path: 'docs',
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl: undefined, // hide edit button
+          versions: {
+            '2.5.x': {
+              label: '2.5.x-2.9.x',
```

Comment:
Let's use `&ndash;` (–) for a better visual effect.
```suggestion
              label: '2.5.x – 2.9.x',
```

Reply:
Done in https://github.com/software-mansion/react-native-reanimated/pull/3589/commits/88c249acf55004907232b609067313d6a0d1462a